main.go

```diff
@@ -955,7 +955,7 @@ func configureSite(ctxt context.Context, ctx *cli.Context, siteNum int, siteStrs
 			console.Fatalln(err)
 		}
 		backend := &Backend{siteNum, endpoint, proxy, &http.Client{
-			Transport: proxy.Transport,
+			Transport: transport,
```

Comment:
While correct - isn't this the same transport instance?
We set `Transport:      transport,` above

Reply:
yes you are right. I found a way to reproduce an issue consistently, this does not fix anything. Closing this.